modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -1988,7 +1997,13 @@ public IgniteInternalFuture<SnapshotPartitionsVerifyTaskResult> checkSnapshot(
 
                 kctx0.task().execute(
                         cls,
-                        new SnapshotPartitionsVerifyTaskArg(grps, metas, snpPath, incIdx, check),
+                        new SnapshotPartitionsVerifyTaskArg(grps,
+                            metas,
+                            snpPath,
+                            metas.values().iterator().next().iterator().next().compress(),
```

Comment:
Do we really need this? Looks like `metas` is here, already, so implementation has access to `compress` flag value.

Reply:
Right. This is redundant. Reverting.